site/_data/github-labels.yaml

```diff
@@ -0,0 +1,308 @@
+      addedBy: label
+      previously:
+        - name: deployment
+    - color: 0052cc
+      description: Issues or PRs related to documentation.
+      name: area/documentation
+      target: both
+      addedBy: label
+      previously:
+        - name: documentation
+    - color: 0052cc
+      description: Issues or PRs related to the HTTPProxy API.
+      name: area/httpproxy
+      target: both
+      addedBy: label
+      previously:
+        - name: HTTPProxy
+    - color: 0052cc
+      description: Issues or PRs related to the IngressRoute API.
+      name: area/ingressroute
+      target: both
+      addedBy: label
+      previously:
+        - name: IngressRoute
+    - color: 0052cc
+      description: Issues or PRs related to the Ingress API.
+      name: area/ingress
+      target: both
+      addedBy: label
+      previously:
+        - name: Ingress
+    - color: 0052cc
+      description: Issues or PRs related to the Gateway (Service APIs working group) API.
+      name: area/service-apis
+      target: both
+      addedBy: label
+      previously:
+        - name: service-apis
+    - color: 0052cc
+      description: Issues or PRs related to the web site.
+      name: area/website
+      target: both
+      addedBy: label
+      previously:
+        - name: website
+    - color: 0052cc
+      description: Issues or PRs related to tests or testing tools.
+      name: area/testing
+      target: both
+      addedBy: label
+      previously:
+        - name: integration-testing
+    - color: 0052cc
+      description: Issues or PRs related to exposing status for APIs.
+      name: area/status
+      target: both
+      addedBy: label
+      previously:
+        - name: status
+    - color: 0052cc
+      description: Issues or PRs related to exposing time series metrics.
+      name: area/metrics
+      target: both
+      addedBy: label
+    - color: 0052cc
+      description: Issues or PRs related to project tools and automation.
+      name: area/tooling
+      target: both
+      addedBy: label
+
+# Blocked.
+    - color: "FF7B7B"
+      description: Categorizes the issue or PR as blocked because there is insufficient information to advance it.
+      name: blocked/needs-info
+      target: both
+      addedBy: label
+      previously:
+        - name: "waiting for info"
+    - color: "FF7B7B"
+      description: Categorizes the issue or PR as blocked because it needs a decision from product management.
+      name: blocked/needs-product
+      target: both
+      addedBy: label
+      previously:
+        - name: "Needs Product"
+    - color: "FF7B7B"
+      description: Categorizes the issue or PR as blocked because it needs a design document.
+      name: blocked/needs-design
+      target: both
+      addedBy: label
+      previously:
+        - name: "needs-design-doc"
+    - color: "FF7B7B"
+      description: Categorizes the issue or PR as blocked because it needs changes in Envoy.
+      name: blocked/needs-envoy
```

Comment:
I'd still leave on blocked upstream, but if pulling out to specific label, we should add a kubernetes one. For example, the ingress v1 work isn't complete upstream and many issues are blocked on that work.

Reply:
Ok, I'll add a Kubernetes one. I think that breaking down "blocked" a bit is worth it, so that we can know what we need to do to unblock something.